static/app/components/avatar/baseAvatar.tsx

```diff
@@ -150,13 +151,15 @@ export {BaseAvatar, type BaseAvatarProps};
 // Note: Avatar will not always be a child of a flex layout, but this seems like a
 // sensible default.
 const StyledBaseAvatar = styled('span')<{
+  hasTooltip: boolean;
   round: boolean;
   suggested: boolean;
 }>`
   flex-shrink: 0;
   border-radius: ${p => (p.round ? '50%' : '3px')};
   border: ${p => (p.suggested ? `1px dashed ${p.theme.subText}` : 'none')};
   background-color: ${p => (p.suggested ? p.theme.background : 'none')};
+  pointer-events: ${p => (p.hasTooltip ? 'none' : 'auto')};
```

Comment:
@narsaynorath Does this work instead, since a few tests depend on the title attribute on line 132